docs/coalton-documentation-guide.md

```diff
@@ -33,6 +33,16 @@ that contains a filling of various sweet or savoury ingredients."
     (Meat String)  "A meat pie with the type of meat."))
 ```
 
+### `define-alias`
+
+`define-alias` allows for the same docstring style as `define`.
+
+```lisp
+(coalton-toplevel
+
```

Comment:
no leading empty line

Reply:
resolved in [5851da6](https://github.com/coalton-lang/coalton/pull/1294/commits/5851da61b96726e42b26b0e2fc066d3136ae4456)